src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollPoll.ts

```diff
@@ -437,7 +437,7 @@ export const transformOktaVerifyEnrollPoll: IdxStepTransformer = ({
 
   const stepper: StepperLayout = {
     type: UISchemaLayoutType.STEPPER,
-    key: 'stepper_' + channelType,
+    key: `stepper_${channelType}`,
```

Comment:
This was pulled in as an automatic lint fix. I'll leave it here but it's unrelated to my actual code changes